[documentation] upd tcp and kafka plugin docs (#1312)
fluent/fluent-bit-docs: pipeline/inputs/tcp.md

The plugin supports the following configuration parameters:

> | Key          | Description                                                                                                                                                                                                                                                    | Default |

Reviewer: Let's start getting these table formats in line.

```suggestion
| Key          | Description | Default |
```

> | ------------ |----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|

Reviewer: This formatting aligns with the suggestion for the previous line.

```suggestion
| ------------ | ----------- | ------- |
```

| Chunk\_Size  | By default the buffer to store the incoming JSON messages, do not allocate the maximum memory allowed, instead it allocate memory when is required. The rounds of allocations are set by _Chunk\_Size_ in KB. If not set, _Chunk\_Size_ is equal to 32 (32KB). | 32      |
| Format       | Specify the expected payload format. It support the options _json_ and _none_. When using _json_, it expects JSON maps, when is set to _none_, it will split every record using the defined _Separator_ (option below).                                        | json    |
| Separator    | When the expected _Format_ is set to _none_, Fluent Bit needs a separator string to split the records. By default it uses the breakline character  (LF or 0x10).                                                                                               |         |
| Source\_Address\_Key| Specify the key where the source address will be injected.                                                                                                                                                                                              |         |

> | Threaded     | Enhance the performance of data ingestion by allowing Fluent Bit to handle incoming data in parallel across multiple dedicated threads.                                                                                                                        | off     |

Reviewer: ```suggestion
| Threaded     | Enhance the performance of data ingestion by letting Fluent Bit handle incoming data in parallel across multiple dedicated threads. | off |
```
